src/tv2-common/segment/context.ts

```diff
@@ -0,0 +1,24 @@
+import { ISegmentUserContext } from 'blueprints-integration'
+import {
+	ExtendedShowStyleContext,
+	ExtendedShowStyleContextImpl,
+	TV2ShowStyleConfig,
+	UniformConfig,
+	VideoSwitcher
+} from 'tv2-common'
+
+export interface ExtendedSegmentContext<BlueprintConfig extends TV2ShowStyleConfig>
```

Comment:
What does `Extended` imply? 
Our config interfaces extends from each other a lot, so what does it mean here that it is `Extended`?

Reply:
I used it to avoid confusing them with core contexts, but it's removed